saas/models/saas_template.py

```diff
@@ -141,3 +141,23 @@ def _rpc_auth(self):
             self.operator_db_name,
             admin_username='admin',
             admin_password=self.password)
+
+    # FIXME: method needs debug and refactor, for example there can be more checks,
+    #  but now tests cannot be reached where this method is called
+    @api.multi
+    def create_db(self, template_db_name, db_name):
+        self.ensure_one()
+        build = self.env['saas.db'].create({
+            'name': db_name,
+            'operator_id': self.operator_id.id,
+            'type': 'build',
+        })
+
+        self.env['saas.log'].log_db_creating(self.operator_db_id)
```

Comment:
It should be in saas.db's create_db, shouldn't it?

Reply:
Ot at least the argument has to be build, not self.operator_db_id

Reply:
> It should be in saas.db's create_db, shouldn't it?

saas_db has own log_db_created(), but I will fix that peace of code